src/org/parosproxy/paros/core/proxy/ProxyThread.java

```diff
@@ -321,14 +325,33 @@ private static void setErrorResponse(HttpMessage msg, String responseStatus, Exc
     private static void setErrorResponse(HttpMessage msg, String responseStatus, Exception cause, String errorType)
             throws HttpMalformedHeaderException {
         StringBuilder strBuilder = new StringBuilder();
-        strBuilder.append(errorType)
-                .append(" [")
-                .append(cause.getClass().getName())
-                .append("]: ")
-                .append(cause.getLocalizedMessage())
-                .append("\n\nStack Trace:\n");
-        for (String stackTraceFrame : ExceptionUtils.getRootCauseStackTrace(cause)) {
-            strBuilder.append(stackTraceFrame).append('\n');
+		
+        if (cause instanceof SSLException) {
+            strBuilder.append("An exception occurred while attempting to connect to: ");
+            strBuilder.append(msg.getRequestHeader().getURI().toString()).append('\n');
+            strBuilder.append("The exception was: " + cause.getMessage());
+            strBuilder.append('\n').append('\n');
+            strBuilder.append(Constant.messages.getString("network.sslconnector.handshake.error.help",
+                    Constant.messages.getString("network.sslconnector.handshake.error.help.url")));
+            
+            LOGGER.warn(strBuilder.toString());

```

Comment:
This doesn't actually work, unless I do a `setLevel()` somewhere. In the constructor? If I set to Level.ERROR how do I ensure that it can be overridden to `Level.DEBUG`?